### PR TITLE
add X-Content-Type-Options: nosniff

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

File: status.json [hunk 1]
```diff
@@ -5011,5 +5011,22 @@
...
+    "name": "X-Content-Type-Options: nosniff header",
+    "category": "Security",
+    "link": "https://fetch.spec.whatwg.org/#should-response-to-request-be-blocked-due-to-nosniff?",
+    "summary": "A header that servers can supply to ensure that any resource it sends has to be exexcuted using the reported Content-Type, and the browser should not attempt to guess or change the file type.",
+    "standardStatus": "Working draft or equivalent",
+    "ff_views": {
+      "text": "Shipped",
+    },
+    "safari_views": {
+      "text": "Shipped",
```
Comment (alrra) [inline status.json:5025]: Remove comma.

File: status.json [hunk 2]
```diff
@@ -5011,5 +5011,22 @@
...
     }
+  },
+  {
+    "name": "X-Content-Type-Options: nosniff header",
+    "category": "Security",
+    "link": "https://fetch.spec.whatwg.org/#should-response-to-request-be-blocked-due-to-nosniff?",
+    "summary": "A header that servers can supply to ensure that any resource it sends has to be exexcuted using the reported Content-Type, and the browser should not attempt to guess or change the file type.",
+    "standardStatus": "Working draft or equivalent",
+    "ff_views": {
+      "text": "Shipped",
```
Comment (alrra) [inline status.json:5022]: Remove comma.

**Review by alrra — COMMENTED** [review]
Merged with some modifications. Thanks, @patrickkettner!

@patrickkettner Can you also [sign the CLA](https://github.com/MicrosoftEdge/Status/pull/559#issuecomment-347734400), thanks!

**Comment by FremyCompany** [comment]
I thought we already supported this header.

**Comment by FremyCompany** [comment]
@patrickkettner @alrra ?

**Comment by FremyCompany** [comment]
https://blogs.msdn.microsoft.com/ie/2008/09/02/ie8-security-part-vi-beta-2-update/

**Comment by patrickkettner** [comment]
there are a number of interop changes on the current spec

**Comment by FremyCompany** [comment]
Ah, that's possible though I haven't heard of any despite following Eric Lawrence on twitter :)

I think you should consult the networking team for input here, but at the very least I think it is misleading to state that we do not ship the feature.

**Comment by alrra** [comment]
> I think you should consult the networking team for input here, but at the very least I think it is misleading to state that we do not ship the feature.

Let's do that.

Note: I've removed the commit for now (will add it back if needed).

**Comment by molant** [comment]
> @patrickkettner Can you also sign the CLA, thanks!

He needs to do something else. I'll follow up via email.

**Pull request closed without merging**